public/app/core/components/Upgrade/UpgradeBox.tsx

```diff
@@ -1,53 +1,35 @@
-import React, { ReactNode, HTMLAttributes } from 'react';
+import React, { HTMLAttributes } from 'react';
 import { css, cx } from '@emotion/css';
 import { Icon, LinkButton, useStyles2 } from '@grafana/ui';
 import { GrafanaTheme2 } from '@grafana/data';
 
 type ComponentSize = 'sm' | 'md';
 
 export interface Props extends HTMLAttributes<HTMLOrSVGElement> {
-  text?: string;
+  featureName: string;
   size?: ComponentSize;
-  children?: ReactNode;
-  secondaryAction?: {
-    url: string;
-    text: string;
-  };
 }
 
-export const UpgradeBox = ({ text, className, children, secondaryAction, size = 'md', ...htmlProps }: Props) => {
+export const UpgradeBox = ({ featureName, className, children, size = 'md', ...htmlProps }: Props) => {
```

Comment:
I don't remember, is there a reason why we don't call the `reportExerimentView` function in this component instead of in every highlight page?

Reply:
Hmm, not sure, maybe we didn't think it'd be used on all the feature pages? But yeah, would makes sense to look into moving the event here instead.